service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -80,11 +81,45 @@
     @Override
     public PolicyEvaluationResult evalPolicy(final PolicyEvaluationRequestV1 request) {
         ZoneEntity zone = this.zoneResolver.getZoneEntityOrFail();
-        PolicyEvaluationRequestCacheKey key = new Builder().zoneId(zone.getName()).policySetId("default")
-                .request(request).build();
-        PolicyEvaluationResult result = this.cache.get(key);
+        String uri = request.getResourceIdentifier();
+        String subjectIdentifier = request.getSubjectIdentifier();
+        String action = request.getAction();
+        List<String> policySetsEvaluationOrder = request.getPolicySetsEvaluationOrder();
+
+        if (uri == null || subjectIdentifier == null || action == null) {
+            LOGGER.error(
+                    String.format(
+                            "Policy evaluation request is missing required input parameters: "
+                                    + "resourceURI=[%s] subjectIdentifier=[%s] action=[%s]",
+                            uri, subjectIdentifier, action));
+
+            throw new IllegalArgumentException("Policy evaluation request is missing required input parameters. "
+                    + "Please review and resubmit the request.");
+        }
+
+        List<PolicySet> allPolicySets = this.policyService.getAllPolicySets();
+
+        if (allPolicySets.isEmpty()) {
+            return new PolicyEvaluationResult(Effect.NOT_APPLICABLE);
+        }
+
+        List<PolicySet> filteredPolicySets = filterPolicySetsByPriority(subjectIdentifier, uri, allPolicySets,
```

Comment:
Refer to comments above, separate user story created. 

https://rally1.rallydev.com/#/59288076735d/detail/userstory/67067764948